styles.scss

```diff
@@ -0,0 +1,127 @@
+@use 'navbar';
+@use 'main-display';
+
+$mainColor: rgb(26, 27, 29);
+$effectColor: rgb(202, 128, 17);
+$mainWhite: #FEFFFF;
```

Comment:
Please be consistent with your approach. RGB or Hex code for colors? choose one and stick to it. There are lots of color tools that can help with conversion.